nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -950,6 +955,45 @@ private void rollback(final boolean penalize, final boolean rollbackCheckpoint)
         resetState();
     }
 
+    private String loggableFlowfileInfo() {
+        final StringBuilder details = new StringBuilder(1024).append("[");
+        final int initLen = details.length();
+        int filesListed = 0;
+        int maxRollbackFlowfilesToLog = 5;
```

Comment:
This seems like a reasonable default for troubleshooting and likely not worth the effort of making it a config option until the utility of this is better understood - but should it be a "constant" somewhere?